pkg/services/ngalert/notifier/channels/teams.go

```diff
@@ -69,10 +69,10 @@ type AdaptiveCardsAttachment struct {
 // AdapativeCard repesents an Adaptive Card.
 // https://adaptivecards.io/explorer/AdaptiveCard.html
 type AdaptiveCard struct {
-	Body    []AdaptiveCardItem `json:"body"`
-	Schema  string             `json:"$schema"`
-	Type    string             `json:"type"`
-	Version string             `json:"version"`
+	Body    []AdaptiveCardItem
+	Schema  string
+	Type    string
+	Version string
```

Comment:
What is the reasoning behind this change?

Reply:
Looks like this was moved to the `MarshalJSON` method of the struct instead

Reply:
It's the same as I did here https://github.com/grafana/grafana/blob/4e17595825d2012c2af8870254f0c4ca24c4437d/pkg/services/ngalert/notifier/channels/teams.go#L171-L185